src/client/pages/Varen/config.ts

```diff
@@ -0,0 +1,41 @@
+import { generatePath } from 'react-router-dom';
+
+import { AppRoutes } from '../../../universal/config/routes';
+import { dateSort } from '../../../universal/helpers/date';
+import { VarenFrontend } from '../../../server/services/varen/config-and-types';
+
+export const displayPropsAanvragen = {
```

Comment:
Er is hier een mooie type voor: `DisplayProps<WithDetailLinkComponent<VarenFrontend>>`